redash/handlers/authentication.py

```diff
@@ -28,6 +28,7 @@ def get_google_auth_url(next_path):
 
 
 def render_token_login_page(template, org_slug, token, invite):
+    error_message = ""
```

Comment:
init `None`

Reply:
lol, thought it would simplify `error_message: str | None = None` to just `error_message: str = ""` in the future but who am i kidding...`render_template()` just doesn't care :joy:

Reply:
> lol, thought it would simplify `error_message: str | None = None` to just `error_message: str = ""` in the future but who am i kidding...`render_template()` just doesn't care: `**context: t.Any` 😂

It's ok!